README.md

```diff
@@ -191,6 +213,50 @@ replacement by a `REPLACE` directive in the same way.
 
 ![Hover over replacement](./assets/superbol-hover-replacement.gif)
 
+## Editing
+
+### Rename Data Items, Sections, and Paragraphs
+
+You can rename any data item by pressing <kbd>F2</kbd> while your
+cursor is positioned on one of its references.  The extension will
+warn you if a reference to the renamed item appears in a copybook (in
+which case the renaming of every reference is not performed).
+
+![Rename Symbol](./assets/superbol-rename-symbol.gif)
+
+![Rename Symbol in Copybook](./assets/superbol-rename-symbol-in-copybook.gif)
+
+Sections and paragraphs of the procedure division can also be renamed
+in the same way.  
+The same limitations as for [Go to References](#go-to-references)
+apply to this feature.
+
+## Exploring the Control-flow
+
+Navigating a graphical representation of a COBOL program's
+control-flow proves invaluable when it comes to decipher its overall
+logic.  To do this, open the command palette (or type
+<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>), and select `SuperBOL:
+Show control-flow` (you can also right click and select `Show
+control-flow` in the menu).  You are then presented with a list of
+portions of program to consider (either the entire program, or
+individiual sections): select one element to see the corresponding
+CFG.
+
+![CFG Explorer](./assets/superbol-cfg-explorer.gif)
+
+Various settings are provided to tune the rendering of CFGs.
+
+![CFG Explorer Collapse Fallthrough](./assets/superbol-cfg-explorer-collapse-fallthrough.gif)
+
+### CFG as an arc diagram
+
+A rendering of CFGs as arc diagrams is also available.  In this
+representation, named sections and paragraphs are laid out vertically,
+and arcs show the direction of control-flow between them.
+
+![CFG Explorer as Arc Diagram](./assets/superbol-cfg-explorer-arc.gif)
+
 ## Debugging
 
 In order to debug a COBOL program, you first need to run a *build
```

Comment:
This section misses things like attaching, setting program/procedure breakpoints, watchpoints, ...

If you change the extension to keep the debugging in its own separate extension (with a dependency in this extension to the debug extension but not vice-versa if the other extension does not use the LSP), then you'd only need a teaser here and can reference to that's extension's README, which should already include parts of this.

Also I'd highly suggest to not only hint to the debug build task, because that is only useful for small/testing programs [
but there it is _very_ useful, especially for beginners]. With bigger environments you'll have either external build flows or (most preferably) done it by `make`.

For this scenario it is important to document which options need to be passed to cobc (I _think_ for GC 3.2+ the `-g` option is enough).


As the coverage handling is also part of that extension, the same applies there - but in any case I'd suggest to have at least a screenshot of that here.

Reply:
I've added basic details about the additional flags passed to `cobc`.

Regarding dependencies, at the moment the debug extension is tuned with a `preLaunchTask` that directs to a build task of the main extension; this is mainly to help newcomers (they can just press <kbd>F5</kbd> to automatically compile and run programs). I'd like to keep the extensions separate, but for now we found no convincing way to do that and keep the aforementioned behavior (it's also still debatable whether this fully automated behavior is really worth the hassle — see also the other debug-related issues).

At the moment I'm trying to do a quick release (long overdue); so aspects mentioned in this comment will be addressed later.

Note we don't rely on the debug extension to show coverage information (there's no reason to have those two features so tied together I think): a dedicated gcov-viewer extension that we found simpler/more maintainable than the coverage-related code of the debug extension is used instead.

Reply:
> a dedicated gcov-viewer extension that we found simpler/more maintainable than the coverage-related code of the debug extension is used instead.

That's good - can you keep that in a separate (depending = automatically installed [as long as you're connected to the internet]) extension or does that needs to "included" as well?

Please create an issue to drop the coverage stuff from the debugging extension (because it is not needed any more and is "too much" there as well).